charts/fission-core/templates/deployment.yaml

```diff
@@ -398,27 +398,49 @@ spec:
         image: {{ include "fission-bundleImage" . | quote }}
         imagePullPolicy: {{ .Values.pullPolicy }}
         command: ["/fission-bundle"]
+        {{- if and .Values.persistence.storageType (.Values.persistence.enabled) }}
+            {{- if eq .Values.persistence.storageType "s3" }}
+        args: ["--storageServicePort", "8000", "--filePath", "/fission", "--storageType", "s3"]
+            {{- end }}
+        {{- else }}
```

Comment:
This is related to another comment in values.yaml - once we are explicit about `storageType` - then arguments being passed to storage service will be fixed.

Reply:
Yes. I've changed and tested for all three cases. 
- Without persistence enabled (tmpDir)
- With persistence( both implicit and explicit `storageType: local`). 
- With `storageType: s3`. 

Please verify.

Reply:
I think in current `values.yaml`  format, your comment is resolve. Can you please confirm.